### PR TITLE
feat: Add directions to access the UI from within HA

### PR DESCRIPTION
**Pull request by blhoward2**
This edit adds directions to add the UI to HA using Lovelace, so that the UI is accessible from within HA itself.

Result:

Screen Shot 2021-01-11 at 10 20 29 AM

### CODE REVIEW

**Comment by robertsLando** [comment]
@blhoward2 I would add those instructions under a new section `##`

**Comment by blhoward2** [comment]
Ok

**Comment by blhoward2** [comment]
@robertsLando How's that?

**Comment by robertsLando** [comment]
@blhoward2 LGTM, could you fix lint issues? `npm run lint-fix`

**Comment by blhoward2** [comment]
I can tonight. I was doing this through the GitHub website so I need to pull it down to do so.

**Comment by robertsLando** [comment]
@zwave-js-bot fix lint

**Comment by robertsLando** [comment]
@AlCalzone Arggg

> Permission to blhoward2/zwavejs2mqtt.git denied to zwave-js-bot

:cry:

**Comment by robertsLando** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
> @AlCalzone Arggg
> 
> > Permission to blhoward2/zwavejs2mqtt.git denied to zwave-js-bot
> 
> 😢

Is there a way for me to give it permission for the future?

**Comment by AlCalzone** [comment]
Interesting... Did you check the option "allow maintainers to make changes"?

**Comment by blhoward2** [comment]
Think so. I’ve never unchecked it.

**Comment by AlCalzone** [comment]
Hmm... think I'll have to investigate. I haven't actually been able to test this on a foreign branch yet.

**Comment by robertsLando** [comment]
@AlCalzone I just forgot to add the bot to my repo. Now it has done something but still not working

File: docs/guide/homeassistant.md [hunk 1]
```diff
@@ -40,6 +40,19 @@ zwavejs2mqtt to resend the cached values when Home Assistant restarts.
 
 zwavejs2mqtt try to do its best to guess how to map devices from Zwave to HASS. At the moment it try to guess the device to generate based on zwave values command classes, index and units of the value. When the discovered device doesn't fit your needs you can you can set custom a `device_class` to values using Gateway value table.
 
+## Accessing zwavejs2mqtt From Within Home Assistant
+
+The zwavejs2mqtt Control Panel UI can be accessed from within Home Assistant by adding the following to a Lovelace dashboard using the Raw Configuration Editor, substituting your IP address.
+
+```
```
Comment (robertsLando) [inline docs/guide/homeassistant.md:47]:
```suggestion
``` yaml
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
Well... it can only fix things that are auto-fixable. Could probably check that and **not** offer the fix if there isn't anything to do.

**Comment by robertsLando** [comment]
Should be fine now. At least I have checked that the bot is working. Thanks @AlCalzone and thanks @blhoward2 for the PR :)

**Comment by AlCalzone** [comment]
Want me to enable the `fix lint` hint comments for this repo too?

**Comment by robertsLando** [comment]
@zwave-js-bot fix lint

**Comment by robertsLando** [comment]
@AlCalzone Yep would be awesome :)

**Comment by AlCalzone** [comment]
Ok, since you're using different linters than me, what's the **exact** check name and which part of the log indicates that there's a lint error?

**Comment by blhoward2** [comment]
@robertsLando @AlCalzone Can anyone invoke the bot or just you guys? For little things I finding pulling/pushing tedious versus just editing in the website and having it automatically create a PR.

**Comment by AlCalzone** [comment]
We both and the respective PR author can do that.

**Comment by blhoward2** [comment]
> We both and the respective PR author can do that. So you should be able to.

Are there directions on how to use it?

**Comment by AlCalzone** [comment]
You comment the following if there is a lint error:
@zwave-js-bot fix lint

A comment explaining that should also appear now (if auto fixing is possible and the script does not fail).

**Comment by chilicheech** [comment]
OK, so maybe I'm just a noob, but it took me a while to figure out how this works. It might be useful to include a link to https://www.home-assistant.io/lovelace/dashboards-and-views/#panel and describe that you need to create a new Lovelace Dashboard, and then go to its raw editor. Also, the instructions say to use the ip address but make no mention of the protocol, http or https. I was putting in just the ip address and it just kept loading hass in an infinite loop. The instructions should say either:
```
url: 'http://[PUT IP ADDRESS HERE]:8091/'
```
or
```
url: 'PUT THE FULL URL WITH THE PORT HERE'
```
or even better, with an example:
```
url: 'http://192.168.1.2:8091'  # replace with the url to your zwavejs2mqtt
```

**Comment by blhoward2** [comment]
Ok, I can make those improvements.

**Pull request merged**